raiden/network/rpc/client.py

```diff
@@ -32,6 +32,18 @@
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def nonce_problem_error_string(version: str)-> str:
```

Comment:
Where is this function used?

Reply:
ops ... ahhahha wanted to have removed this.

Reply:
dissapeared from history :octocat: